### ISSUE TITLE
The performance of the new design is quite bad (Lighthouse)

### ISSUE THREAD

**Issue by jankeromnes** [gitpod-io/website#699]
<img width="583" alt="Screenshot 2020-07-16 at 12 52 18" src="https://user-images.githubusercontent.com/599268/87662902-431ec080-c763-11ea-89d0-2feb7915b141.png">

**Comment by jankeromnes** [comment]
Many thanks for catching and reporting this @samuelpath! 🙏

Some more relevant findings:

> 16 MB of resources for a static landing page

> 22.34 seconds to finish loading the page on my latest generation MacBook Air with fiber

**Comment by jankeromnes** [comment]
It seems that most of the performance degradation (JS size & loading time) is actually caused just by the embedded Tweets (thanks again @samuelpath for the investigation! 💯)

So, likely the biggest optimisation we could make is to fetch the Tweets at build-time, and to display the contents ourselves with minimal front-end code, instead of relying on Twitter's heavyweight embedding scripts at load-time.

The next optimisation would probably be image sizes, and I've already requested a new PR from @ImgBotApp.

**Comment by nisarhassan12** [comment]
Regarding image sizes, I think it would be a lot better if we were to use `gatsby-image` which uses `gatsby-plugin-sharp` which does the transformations. It solves a lot of other problems as well https://www.gatsbyjs.org/packages/gatsby-image/

**Comment by ChristinFrohne** [comment]
Google PageSpeed might also have some helpful information to fix it: https://developers.google.com/speed/pagespeed/insights/?url=www.gitpod.io

**Comment by nisarhassan12** [comment]
We got a 5% score improvement with the self-build tweets. I believe the score will jump up once #826 is merged. 
![image](https://user-images.githubusercontent.com/46004116/103158534-ae73c080-47e0-11eb-91b6-c1afe9d5e1c1.png)
